marklogic-data-hub/src/test/java/com/marklogic/hub/dhs/installer/deploy/CopyQueryOptionsCommandTest.java

```diff
@@ -21,11 +21,11 @@ void test() throws Exception {
         // clear user modules to ensure extra options aren't left from other tests
         clearUserModules();
 
-        final String groupName = getHubConfig().getAppConfig().getGroupName();
-        final String stagingServerName = getHubConfig().getHttpName(DatabaseKind.STAGING);
-        final String jobsServerName = getHubConfig().getHttpName(DatabaseKind.JOB);
+        final String groupName = adminHubConfig.getAppConfig().getGroupName();
```

Comment:
This one is still incorrect - "adminHubConfig" was removed on 5.4-develop and replaced with "getHubConfig()".

Reply:
@rjrudin fixed and pushed this additional one, please review.